### PR TITLE
Autolathe rework, All trash recyclable

### PR DESCRIPTION
**Pull request by Aurelien3189**
Reworks the autolathe list, both the organisation of the items (ammo) and most of the costs, along with a few additions.

### Autolathe list:
-Ammo reorganised completely as opposed to having different shenanigans mixed around, relabelled as [weapon type] [(caliber)], [(type)]... Ex: smg magazine (7mm), (practice)
-Added ammo variants (rubber and practice) to those that exist, as well as a few new rounds, including the prototype smg ammo (4mm) and the net shell. Fixes beanbag shells not appearing in the list
-replaced printing individual shotgun shells with printing the respective shotgun holders (4 shells at a time)
-Added a few items to the general list, including duct tape, compact fire extinguisher, and maglight.
image
image

### Matter costs:
-tweaked shotgun shell holders w/ammo to be as expensive as 4 of their respective shells and the shotgun shell holder
-surgical tool prices have been slashed to costs resembling other tools (~20% cost avg)
-duct tape given a new price (1000 plastic, 100 aluminum) to be printable
-ALL TRASH can now be recycled (default 50 plastic, variants for glass and aluminum items)
-All bullet casings can now be recycled
image
image

Please look through it and let me know of any necessary tweaks or requests to this PR. Mostly started this PR to rework the crazy ammo order, but it escalated into a more full rework.

🆑 :tweak: reorganised all ammo, grouping them by category and adding existing variants. ex: pistol magazine (10mm), (extended), (practice)
:tweak: added net shells, prototype smg ammo, duct tape and the compact extinguisher to the autolathe
:tweak: matter tweaks: all trash can be recycled for plastic or other material along with bullet casings. Scalpel no longer costs a stack of metal, and other cost tweaks.
/🆑

### CODE REVIEW

**Comment by Aurelien3189** [comment]
I need some help here, rather some direction would be nice to see what's wrong

File: code/game/objects/items/weapons/extinguisher.dm [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	var/sprite_name = "fire_extinguisher"
 
 /obj/item/weapon/extinguisher/mini
-	name = "fire extinguisher"
+	name = "compact fire extinguisher"
```
Comment (Flying-loulou) [inline code/game/objects/items/weapons/extinguisher.dm:26]: Not sure that name change was necessary, but I don't think that's an issue either.

Comment (Aurelien3189) [reply code/game/objects/items/weapons/extinguisher.dm:26]: It had the same name as the regular fire extinguisher, so you'd have what looked like "Fire extinguisher" appear twice in the list, right next to eachother, with one half the cost of the other.

**Comment by bearrrrrrrr** [comment]
Looks good! The only thing I'll say is that you definitely want to increase the cost on the .454 speedloader by a whole lot if it's the cylinders that the Captain's magnum uses. IIRC Carl said he put the cost to like...that of an RCD, at one point. Stuff's real powerful.

**Pull request merged**